src/rgw/rgw_compression.cc

```diff
@@ -79,7 +79,6 @@ int RGWGetObj_Decompress::handle_data(bufferlist& bl, off_t bl_ofs, off_t bl_len
   bufferlist out_bl, in_bl, temp_in_bl;
   bl.copy(bl_ofs, bl_len, temp_in_bl); 
   bl_ofs = 0;
-  int r;
```

Comment:
I think that simple initialization "int r = 0; " would be better.
It will set value that is to be returned, if no actions with next->handle_data are executed.
Otherwise, we just pass value.

Reply:
ok!